### PR TITLE
Upgrade to dropwizard 2.1.6

### PR DESCRIPTION
**Pull request by karussell**
This worked without a major problem.

### CODE REVIEW

**Comment by easbar** [comment]
With 2.1.4 at least the build pipelines seem to be green.

File: web/src/test/java/com/graphhopper/application/resources/IsochroneResourceTest.java [hunk 1]
```diff
@@ -274,11 +274,11 @@ public void requestNotANumber() {
         Response response = clientTarget(app, "/isochrone?profile=fast_car&point=42.531073,1.573792&time_limit=wurst")
                 .request().buildGet().invoke();
 
-        assertEquals(400, response.getStatus());
+        assertEquals(404, response.getStatus());
         JsonNode json = response.readEntity(JsonNode.class);
         String message = json.path("message").asText();
 
-        assertEquals("query param time_limit is not a number.", message);
+        assertEquals("HTTP 404 Not Found", message);
```
[the remark this answers is not in the document]
Comment (easbar) [reply web/src/test/java/com/graphhopper/application/resources/IsochroneResourceTest.java:281]: Shall we update anyway? Btw I just upgraded to 2.0.34: #2682

[the remark this answers is not in the document]
Comment (easbar) [reply web/src/test/java/com/graphhopper/application/resources/IsochroneResourceTest.java:281]: Ok, we could also wait a few months and see if the issue gets fixed in jersey then: https://github.com/eclipse-ee4j/jersey/issues/4790#issuecomment-1299951555

[the remark this answers is not in the document]
Comment (michaz) [reply web/src/test/java/com/graphhopper/application/resources/IsochroneResourceTest.java:272]: Reverting #2574 fixes this

**Pull request merged**